test/unit_tests/braket/ocean_plugin/conftest.py

```diff
@@ -39,6 +40,11 @@ def shots():
 
 @pytest.fixture
 def service_properties():
+    return {"shotsRange": [0, 10]}
+
+
+@pytest.fixture
+def service_properties_list():
```

Comment:
Could you please provide some context on why this is needed ?

Reply:
The input expected is the array input but the comparison done later is converted to the list.